### ISSUE TITLE
Request: add dark mode to web interface

### ISSUE THREAD

**Issue by Hackashaq666** [owntone/owntone-server#1470]
Anyone have the skills to add a dark mode to the web interface? Thank you.

**Comment by natemacinnes** [comment]
@Hackashaq666 I have a working prototype, I'll submit a PR for consideration.

**Comment by natemacinnes** [comment]
![owntone-dark-mode](https://github.com/owntone/owntone-server/assets/1396497/bb580a16-6182-4010-9f6b-b3e2fc390fa5)

**Comment by hacketiwack** [comment]
@natemacinnes Thanks for the working prototype.
@Hackashaq666, what is the expected behaviour? How should the dark mode be activated: with an option in the settings or automatically based on the operating system settings?

**Comment by Porco-Rosso** [comment]
I would suggest the default to be system settings, but if changed manually, the selected option should be preserved via cookie.

**Comment by flintstones-fred** [comment]
I was wondering if there was any activity around this?

**Comment by hacketiwack** [comment]
@flintstones-fred this work item is not forgotten. I will tackle that one once we have the lyrics feature integrated.

**Comment by flintstones-fred** [comment]
Thank you @hacketiwack . Is there a roadmap for owntone? I have been really enjoying this service, especially how well it handles multi-room audio (based on the speakers you have). I would like to follow the progress along. Thanks.

**Comment by hacketiwack** [comment]
@flintstones-fred there is no official roadmap for OwnTone. @ejurgensen is the original owner of OwnTone, and could potentially better answer this question.

**Comment by ejurgensen** [comment]
Yes, there isn't really a roadmap. It's just what myself and @hacketiwack find worth working on, and of course if interesting contributions come in. Personally, I would like to do some further Airplay improvements, but I can't say when (or if).

**Comment by hacketiwack** [comment]
I added a dark mode to the UI. You have to set the appearance to dark in your OS to get the dark mode in OwnTone.

**Comment by cdlenfert** [comment]
Works for me with my system preference set (on iOS). Very cool!!

It would be nice to include a toggle in Owntone's general settings: http://owntone.local:3689/#/settings/webinterface, but certainly a "nice to have".

**Comment by hacketiwack** [comment]
@cdlenfert, thanks 😊
Yes, I will implement a setting in the future.

**Comment by ejurgensen** [comment]
Just tried the new dark mode and I think it's great - nice job @hacketiwack!

**Comment by hacketiwack** [comment]
@ejurgensen Thanks!
There are 2-3 glitches I need to correct though. I'll correct them in the coming days.

**Comment by pisabird** [comment]
@hacketiwack 

Is the toogle already implented? Because I don`t want that the UI use my darkmode settings from iOS.

**Comment by pisabird** [comment]
I am fine,
I changed via CSS

background-color: white;

**Comment by hacketiwack** [comment]
Not yet unfortunately.

**Comment by pisabird** [comment]
which version is the last without darkmode?

**Comment by hacketiwack** [comment]
It is now possible to change the appearance of the UI in the settings with the latest commit.
